docs/tutorials/index.md

```diff
@@ -0,0 +1,11 @@
+# Tutorials
+
+This is the totorials section for beginners.
```

Comment:
typo in tutorials

Reply:
#263 should automatically flag these for us.